src/disclosures/js/utils/query-handler.js

```diff
@@ -8,7 +8,13 @@ var stringToNum = require( './handle-string-input.js' );
  * @returns {object} - An object containing key-value pairs from the query
  */
 function queryHandler( queryString ) {
-  var valuePairs = {};
+  var valuePairs = {
```

Comment:
`gulp lint` is throwing the following warning for this group of lines:
 
> Properties shouldn't be quoted as all quotes are redundant 

Since we're following this rule in the rest of the codebase, can you remove the quotes here as well?